pkg/cmd/cli/install/install.go

```diff
@@ -101,6 +102,7 @@ func (o *InstallOptions) BindFlags(flags *pflag.FlagSet) {
 	flags.Var(&o.Plugins, "plugins", "Plugin container images to install into the Velero Deployment")
 	flags.BoolVar(&o.CRDsOnly, "crds-only", o.CRDsOnly, "only generate CustomResourceDefinition resources. Useful for updating CRDs for an existing Velero install.")
 	flags.StringVar(&o.CACertFile, "cacert", o.CACertFile, "file containing a certificate bundle to use when verifying TLS connections to the object store. Optional.")
+	flags.StringVar(&o.Features, "features", o.Features, "comma separated list of Velero feature flags to be set on the Velero deployment")
```

Comment:
I think we probably want to set these on the restic DaemonSet too, if we're creating it - @nrb WDYT?

Reply:
Yep, I agree.

Reply:
I can do that. I'll need to confirm that this doesn't trigger the `unbound flags` error

Reply:
done! Testing also looked good. No `unknown flag` errors